### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by pallavipannu**
i tried the example with the latest version, some minor fixes were there.
@songole @gaurav-oc Please review and merge

### CODE REVIEW

File: insurance/resources.ipynb [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 15,
```
Comment (gaurav-oc) [inline insurance/resources.ipynb]: Please clear this execution count

File: insurance/resources.ipynb [hunk 2]
```diff
@@ -84,9 +84,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 16,
```
Comment (gaurav-oc) [inline insurance/resources.ipynb]: Clear this as well and output

**Review by gaurav-oc — COMMENTED** [review]
Please remove this cell as well https://github.com/oneconvergence/dkube-examples/blob/cba854430b1b8cd0a3b421fe278f25ba167c64d7/insurance/resources.ipynb?short_path=a6a7b9d#L341 i forget to remove it while improvement.

**Comment by pallavipannu** [comment]
@songole @gaurav-oc updated the pr.

**Comment by gaurav-oc** [comment]
Seems to be perfect merging it @songole

**Pull request merged**